examples/permissions/set_permissions_example.py

```diff
@@ -27,7 +27,7 @@
 Set and query permissions
 =========================
 
-Example script to set and query permissions on files and
```

Comment:
This ReStructuredText formatting within the script also looks odd and perhaps should be changed:
    ###################################
    # Create a DataTransferApi instance
    # =================================

Reply:
Addressing here: https://github.com/ansys/pyhps-data-transfer/pull/200/files